request_a_govuk_domain/request/models/organisation.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+
+class Organisation(models.Model):
+    name = models.CharField()
+
+
+class RegistrantTypeChoices(models.TextChoices):
+    central_goverment = "Central government department or agency"
```

Comment:
Another minor point is that views have used  "central_gov"  https://github.com/co-cddo/domains-register-a-govuk-domain/blob/main/request_a_govuk_domain/request/views.py#L92 in some places for logical flows , so they will need updating.